xla/service/gpu/runtime/command_buffer_thunk_test.cc

```diff
@@ -1473,6 +1473,8 @@ class CmdBufferTest : public HloTestBase {
  public:
   DebugOptions GetDebugOptionsForTest() const override {
     DebugOptions debug_options = HloTestBase::GetDebugOptionsForTest();
+    debug_options.set_xla_gpu_require_exclusive_lock(true);
```

Comment:
Instead of adding a new test, this now switched the CmdBufferTest to the new behavior, leaving the old behavior untested.

@ezhulenev Is this desirable? If not, should this be a parametric test or do would you suggest a different test entirely?

Reply:
Let's parameterize it. It's the only low-level command buffer thunk test. It's true that the default path is tested in many other places, but if something will break it's a lot easier to debug this one.

Reply:
@shawnwang18 Please address this comment and the PR would be good to merge.